exercises/all-your-base/all_your_base_test.rb

```diff
@@ -0,0 +1,114 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'all_your_base'
+
+class AllYourBaseTest < Minitest::Test
+  def test_single_bit_to_one_decimal
+    assert_equal [1], BaseConverter.new.convert(2, [1], 10)
+  end
+
+  def test_binary_to_single_decimal
+    skip
+    assert_equal [5], BaseConverter.new.convert(2, [1, 0, 1], 10)
```

Comment:
It would be a method in the `AllYourBaseTest` class that would return an array of values as an input to the converter. 
On reflection **I don't think it's a good idea here**, since you're already getting the input arrays from the json file.
I'm just lazy and would rather type "101" than "[1, 0, 1]"